### PR TITLE
Button Type scene

### PR DESCRIPTION
**Pull request by grzegorzdec**
ezgif-5-531f2b907f31

### CODE REVIEW

File: src/test_scenes/sceneButtonType.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { LinearLayout, Button } from 'magic-script-components';
+
+class SceneButtonType extends React.Component {
+	constructor(props) {
+		super(props);
+	}
+
+	render() {
+		return (
+			<view localPosition={this.props.localPosition}>
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneButtonType.js]: Let's use `View` instead of `view`.

File: src/test_scenes/sceneButtonType.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { LinearLayout, Button } from 'magic-script-components';
+
+class SceneButtonType extends React.Component {
+	constructor(props) {
+		super(props);
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneButtonType.js]: We don't need this constructor.

File: src/test_scenes/sceneButtonType.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+					<Button text='text' height={0.15} />
+					<Button iconType='clock' text='text' height={0.15} />
+					<Button type='text-with-icon' labelSide='right' iconType='clock' text='text' height={0.15} />
+					<Button type='icon-with-label' labelSide='top' iconType='clock' text='top' height={0.15} />
+					<Button type='icon-with-label' labelSide='right' iconType='clock' text='right' height={0.15} />
+					<Button type='icon-with-label' labelSide='bottom' iconType='clock' text='bottom' height={0.15} />
+					<Button type='icon-with-label' labelSide='left' iconType='clock' text='left' height={0.15} />
+					<Button type='icon' labelSide='left' iconType='clock' text='text' height={0.15} />
+					<Button type='text' labelSide='left' iconType='clock' text='text' height={0.15} />
+				</LinearLayout>
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneButtonType.js]: Can we also add:
- iconColor
- iconSize
- iconPath
?

**Review by pleszkiewicz — APPROVED** [review]
Code: +0.99 (please use View and remove irrelevant code)

File: src/test_scenes/sceneButtonType.js (hunk 3 above)
Comment (grzegorzdec) [reply src/test_scenes/sceneButtonType.js]: I'd like to keep this scene simple and focused only on button type possibilities, is it fine?

**Pull request merged**